crypto/src/main/kotlin/web5/sdk/crypto/Ed25519.kt

```diff
@@ -156,8 +190,8 @@ public object Ed25519 : KeyGenerator, Signer {
    * @param key The [JWK] to validate.
    * @throws IllegalArgumentException if the key is not of type [OctetKeyPair] or if the key type is not [KeyType.EC].
    */
-  public fun validateKey(key: JWK) {
-    require(key is OctetKeyPair) { "private key must be an Octet Key Pair (kty: OKP)" }
+  private fun validateKey(key: JWK) {
+    require(key is OctetKeyPair) { "key must be an Octet Key Pair (kty: OKP)" }
     require(key.keyType == keyType) { "private key key type must be OKP" }
```

Comment:
Since the `validateKey()` function is shared by both `validatePrivateKey()` and `validatePublicKey()` should this error not reference "private" since it may be raised when checking whether a public key's `kty` is `OKP`?

```suggestion
    require(key.keyType == keyType) { "key type must be OKP" }
```